library/oneview_volume_facts.py

```diff
@@ -124,9 +136,9 @@
 class VolumeFactsModule(OneViewModuleBase):
     def __init__(self):
         argument_spec = dict(
-            name=dict(required=False, type='str'),
-            options=dict(required=False, type='list'),
-            params=dict(required=False, type='dict'),
+            name=dict(type='str'),
```

Comment:
Lines 139-141 could probably fit into line 138 without issues to readability/line length.